frontend/components/navbar/ProfilePic.tsx

```diff
@@ -1,9 +1,9 @@
 import { handleSignOut } from "@/services/src/auth/authService";
 import { auth, storage } from "@/services/src/firebase";
 import { sleep } from "@/utilities/sleepUtil";
-import { Menu, Transition } from "@headlessui/react";
+import { Menu, MenuButton, MenuItem, MenuItems, Transition } from "@headlessui/react";
```

Comment:
Module '"@headlessui/react"' has no exported member 'MenuItems' 
same error for MenuButton, MenuItem, MenuItems
not sure whats going on here but verify if working for u and get back to me

Reply:
doing an npm i in the frontend folder fixed this error for me.

Also, it seems this may also cause some issues when running npm run dev with an older version of node (12.22.0)
Use nvm to upgrade version of node.